### ISSUE TITLE
Packer data blocks glow red

### ISSUE THREAD

**Issue by garthk** [4ops/vscode-language-packer#3]
This glows red:

```hcl
data "amazon-ami" "amazon_source_ami" {
  filters     = { name = "amzn2-ami-hvm-*-x86_64-ebs" }
  most_recent = true
  owners      = ["amazon"]
  region      = var.AWS_REGION
}
```

Strikes me you're specially matching on some block identifiers _eg._ `build`, `locals`, and `source` when HCL2 permits any identifier to introduce the block. From the [spec](https://github.com/hashicorp/hcl2/blob/master/hcl/hclsyntax/spec.md#structural-elements):

> ```ebnf
> ConfigFile   = Body;
> Body         = (Attribute | Block | OneLineBlock)*;
> Attribute    = Identifier "=" Expression Newline;
> Block        = Identifier (StringLit|Identifier)* "{" Newline Body "}" Newline;
> OneLineBlock = Identifier (StringLit|Identifier)* "{" (Identifier "=" Expression)? "}" Newline;
> ```

If you leave unknown block identifiers red but at least parse the rest, any other block identifiers you missed won't have such an impact.

**Comment by asciifaceman** [comment]
@garthk have you found an alternative plugin? This author doesn't seem to be around anymore

**Comment by garthk** [comment]
Not yet, sorry.

**Comment by asciifaceman** [comment]
@garthk it's not a perfect solution, but the Terraform plugin itself handles the packer hcl files "ok"

**Comment by 4ops** [comment]
Fixed in release [v0.3.0](https://github.com/4ops/vscode-language-packer/releases/tag/v0.3.0)
